### PR TITLE
Add Selenium and Firefox to vm to power Behat tests

### PR DESCRIPTION
**Pull request by arknoll**
Relates to https://github.com/geerlingguy/drupal-vm/issues/137

### CODE REVIEW

File: example.config.yml [hunk 1]
```diff
@@ -155,3 +156,6 @@ php_xdebug_default_enable: 0
 solr_version: "4.10.4"
 solr_xms: "64M"
 solr_xmx: "128M"
+
+# Selenium configuration
+server_version: 2.46.0
```
Comment (geerlingguy) [inline example.config.yml]: Can this be updated (upstream) to a variable with the name of the role prefixed (e.g. `selenium_server_version`)? Similar to Drupal modules, it's best to use a prefixed variable name so there are no conflicts between roles (and to clarify what, exactly, the variable is used for in a large config file).

File: provisioning/requirements.txt [hunk 2]
```diff
@@ -21,3 +21,4 @@ geerlingguy.repo-remi
 geerlingguy.security
 geerlingguy.solr
 geerlingguy.varnish
+arknoll.selenium
```
Comment (geerlingguy) [inline provisioning/requirements.txt]: I've been adding roles in alpha order - can you stick your role at the top of the file instead? (OCD, I know... ;).

File: example.config.yml (hunk 1 above)
Comment (geerlingguy) [inline example.config.yml]: Actually, it looks like the variable _is_ `selenium_version` upstream... is that correct?

**Comment by geerlingguy** [comment]
One other question: Would you be willing to (or would you accept a PR from me to) make this work on RHEL/CentOS? Currently all other configuration in the VM works across platforms, so people can switch to other base boxes if they would like, and I don't want to take away that ability if they want to use selenium.

File: provisioning/requirements.txt (hunk 2 above)
Comment (arknoll) [inline provisioning/requirements.txt]: Fixed.

File: example.config.yml (hunk 1 above)
Comment (arknoll) [inline example.config.yml]: yeah, it should have been selenium_version. Fixed.

**Comment by arknoll** [comment]
@geerlingguy let me give it a shot to get it working on RHEL/CentOS.

**Comment by geerlingguy** [comment]
@arknoll - Great! let me know when that's done, and I'll un-postpone this PR.

**Comment by arknoll** [comment]
@geerlingguy Done and tested on CentOS https://galaxy.ansible.com/list#/roles/4340.

**Comment by geerlingguy** [comment]
Great! I'll merge and do some more testing tonight.

**Comment by geerlingguy** [comment]
Verified that behat/selenium/firefox/etc. works great across all supported CentOS and Ubuntu versions.

**Pull request merged**